pyvisa_py/attributes.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+"""Additional Attributes for specific use with the pyvisa-py package.
+
+For additional information and VISA attributes see pyvisa.constants
+
+:copyright: 2014-2020 by PyVISA-py Authors, see AUTHORS for more details.
+:license: MIT, see LICENSE for more details.
+"""
+
+from pyvisa import constants
+from pyvisa.attributes import BooleanAttribute
+
+
+class AttrVI_ATTR_TCPIP_KEEPALIVE(BooleanAttribute):
```

Comment:
I noticed you previously experimented with different solutions. Was there no way to rely on inheritance to get the values set in pyvisa ? Alternatively you could copy them over from pyvisa to avoid possible divergence in the future.

Reply:
To be honest I tried a lot and now that I am thinking about it I'm not sure why I'm not inheriting from the attribute directly. I will try that out as it would cut down the lines of code significantly. An changes from pyvisa would be handed down.